Aztec/Classes/Public/TextKit/TextView.swift

```diff
@@ -418,9 +418,10 @@ public class TextView: UITextView {
         attachment.image = image
 
         // Inject the Attachment and Layout
-        let insertionRange = NSMakeRange(position, 0)
+        let insertionRange = NSMakeRange(position, 1)
```

Comment:
This make the image "overwrite" any characters that is next the insertion point.